runtime/startup_aux.c

```diff
@@ -103,6 +103,9 @@ void caml_parse_ocamlrunparam(void)
       case 'c': scanmult (opt, &params.cleanup_on_exit); break;
       case 'd': scanmult (opt, &params.max_domains); break;
       case 'e': scanmult (opt, &params.runtime_events_log_wsize); break;
+      case 'F':
+        // scanned value is positive in case we declare the warning
+        scanmult (opt, (uintnat*)&caml_float_array_alloc_warning); break;
```

Comment:
Nit, but I would remove the comment; it is not clear what it is referring to without more context (which warning?).
```suggestion
      case 'F': scanmult (opt, (uintnat*)&caml_float_array_alloc_warning); break;
```